test/e2e/safari/webview/special-caps-specs.js

```diff
@@ -1,50 +0,0 @@
-import desired from './desired';
```

Comment:
I just moved them. The Safari tests have been epically flakey in this driver, and I can't figure out why. But since the tests themselves are weird, I though it might be the order.